Sources/Swift/Protocol/SentryLogAttribute.swift

```diff
@@ -0,0 +1,26 @@
+@objcMembers
+public final class SentryLogAttribute {
```

Comment:
Was considering using an enum with associated values, but uses a class instead to be compatible with objc.

Reply:
Since we are using the Swift type, i opted to use the enum with associated value approach. That way we only have the closed set of supported values. We can extend it with list types as per docs by adding the enum cases.